frontend/src/components/Sidepanel/Panels/Info/Info.js

```diff
@@ -47,6 +53,7 @@ const Info = () => {
 
     return map
   }, [states, alphabet, resolvedGraph])
+  console.log(transitionMap);
```

Comment:
naughty naughty

Reply:
what do you mean by this 👀 nothing happened, nothing at all.